// react-media-previewer/dist/utils/index.js
const isExtInclude = (filename, extList) => {
    const dotPos = filename.lastIndexOf(".");
    if (dotPos >= 0) {
        const ext = filename.substr(dotPos);
        return extList.includes(ext.toLowerCase());
    }
    return false;
};
const wordExts = [".docx", ".doc", ".rtf", ".html", ".odt"];
const excelExts = [
    ".xlsx",
    ".xls",
    ".csv",
    ".xlsb",
    ".xlsm",
    ".xlt",
    ".xltm",
    ".xltx",
];
const pptExts = [".ppt", ".pptx"];
export const isImageFile = (contentType) => {
    return contentType.startsWith("image");
};
export const isAudioFile = (contentType) => {
    return contentType.startsWith("audio");
};
export const isVideoFile = (contentType) => {
    return contentType.startsWith("video");
};
export const isPDFFile = (contentType) => {
    return contentType.startsWith("application/pdf");
};
export const isOfficeWordFile = (filename) => {
    return isExtInclude(filename, wordExts);
};
export const isOfficeExcelFile = (filename) => {
    return isExtInclude(filename, excelExts);
};
export const isOfficePPTFile = (filename) => {
    return isExtInclude(filename, pptExts);
};
